src/app/wallet/wallet/history/history.component.html

```diff
@@ -1,24 +1,50 @@
-<app-header>
-  <div class="container-fluid">
-    <!--ul class="header-nav">
-      <li [class.active]="category == 'all'"><a href="javascript:void(0)" (click)="filterByCategory('all')">All</a></li>
-      <li [class.active]="category == 'send'"><a href="javascript:void(0)" (click)="filterByCategory('send')">Send</a></li>
-      <li [class.active]="category == 'receive'"><a href="javascript:void(0)" (click)="filterByCategory('receive')">Receive</a></li>
-      <li [class.active]="category == 'stake'"><a href="javascript:void(0)" (click)="filterByCategory('stake')">Stake</a></li>
-      <li [class.active]="category == 'orphaned_stake'"><a href="javascript:void(0)" (click)="filterByCategory('orphaned_stake')">Orphan</a></li>
-    </ul>
-    <form class="header-form pull-right">
-      <input type="text" placeholder="Search history">
-      <button class="btn" type="submit" name="button">
-        <span class="fa fa-search"></span>
-      </button>
-    </form-->
-  </div>
-</app-header>
-
-<div class="container-fluid container-block">
-  <transaction-table
-    [display]="{'header': false, 'internalHeader': true, 'pagination': true, 'category': true, 'expand': true}">
-  </transaction-table>
-</div>
+<app-header class="disable-select"></app-header>
+
+<div fxLayout="row" fxLayoutGap="30px" class="container-block">
+
+  <div class="filter" fxFlex="0 0 200px">
+
+    <div class="section filter-by-type">
+      <div class="subtitle">Filter by type</div>
+      <mat-card class="filter-card">
+        <mat-selection-list #types dense class="filter-list no-padding">
```

Comment:
@allienworks this seems to trigger an error in the console, can you look into it ?

Reply:
https://github.com/angular/material2/blob/master/src/demo-app/list/list-demo.html